torch/csrc/Module.cpp

```diff
@@ -1238,6 +1256,11 @@ static PyMethodDef TorchMethods[] = { // NOLINT
      METH_NOARGS,
      nullptr},
     {"_set_sdp_use_math", THPModule_setSDPUseMath, METH_O, nullptr},
+    {"_get_cudnn_sdp_enabled",
```

Comment:
There should be another bypinded struct in this file that looks like: 

``` C++
  py::enum_<sdp::SDPBackend>(
      py_module,
      "_SDPBackend",
      "Enum class for the scaled dot product attention backends\n\n... warning:: This class is in beta and subject to change.")
      .value("ERROR", sdp::SDPBackend::error)
      .value("MATH", sdp::SDPBackend::math)
      .value("FLASH_ATTENTION", sdp::SDPBackend::flash_attention)
      .value("EFFICIENT_ATTENTION", sdp::SDPBackend::efficient_attention);
      ```
      
      
      can you also update the enum here?